.github/workflows/r-python-interop-testing.yml

```diff
@@ -27,54 +27,19 @@ jobs:
       - name: Checkout TileDB-SOMA
         uses: actions/checkout@v4
         with:
-          fetch-depth: 0  # ensure we get all tags to inform package version determination
+          fetch-depth: 0 # ensure we get all tags to inform package version determination
 
       - name: Bootstrap
         run: cd apis/r && tools/r-ci.sh bootstrap
 
       - name: Dependencies
         run: cd apis/r && Rscript -e "remotes::install_deps(dependencies = TRUE, upgrade = FALSE)"
 
-      - name: CMake
-        uses: lukka/get-cmake@latest
+      - name: Build Package
+        run: cd apis/r && R CMD build --no-build-vignettes --no-manual .
 
-      - name: MkVars
-        run: mkdir ~/.R && echo "CXX17FLAGS=-Wno-deprecated-declarations -Wno-deprecated" > ~/.R/Makevars
-
-      # Uncomment these next two stanzas as needed whenever we've just released a new tiledb-r for
-      # which source is available but CRAN releases (and hence update r2u binaries) are not yet:
-      #
-      # IMPORTANT: these two stanzas should remain uncommented _only_ during the propagation time
-      # between (a) publication of source for a new TileDB-R _upon which_ TileDB-SOMA depends in its
-      # apis/R/DESCRIPTION file and (b) appearance of binaries.
-      #
-      # Please see https://github.com/single-cell-data/TileDB-SOMA/wiki/Branches-and-releases which
-      # is crucial for anyone doing releases of TileDB-SOMA.
-      #
-      # Please edit both files in the same way:
-      # * r-ci.yml
-      # * r-python-interop-testing.yml
-      #
-      # Do not remove these comments until such time as we have eliminated our dependency on
```

Comment:
Please don't remove this comment block until we've removed our dependency on the TileDB-R package (which is till WIP).

Reply:
added back